interface/interface.dm

```diff
@@ -4,8 +4,12 @@
 	set desc = "Type what you want to know about.  This will open the wiki in your web browser."
 	set hidden = 1
 	if(config.wikiurl)
-		if(alert("Open the wiki in your browser?", null, "Yes", "No") == "Yes")
+		var/query = stripped_input(src, "Enter Search:", "Wiki Search", "Homepage")
```

Comment:
Add a parameter to this verb which can skip the popup entirely and take straight to the homepage, since that what the Wiki button in the changelog does. See https://github.com/ParadiseSS13/Paradise/blob/3ed5f7caf547a0b9513a87e6f8308ed3514e2b5e/code/controllers/subsystem/changelog.dm#L255

They would have to press buttons in 2 different dialogue boxes, which is awful

Reply:
Deleted the first button, kept the second (since it allows all 3 options, whereas the 1st one did not).